catalog/java/io/material/catalog/slider/SliderScrollContainerDemoFragment.java

```diff
@@ -43,10 +44,11 @@ public View onCreateDemoView(
     for (int i = 0; i < 50; i++) {
       Slider slider = new Slider(layoutInflater.getContext());
       slider.setValueTo(11f);
+      slider.setOrientation(SliderOrientation.VERTICAL);
```

Comment:
Could you create a separate demo for this?

Reply:
I created it.